code/lib/preview-web/src/render/StandaloneDocsRender.ts

```diff
@@ -96,12 +98,17 @@ export class StandaloneDocsRender<TFramework extends AnyFramework> implements Re
     };
 
     this.rerender = async () => renderDocs();
-    this.teardown = async ({ viewModeChanged }: { viewModeChanged?: boolean } = {}) => {
+    this.teardownRender = async ({ viewModeChanged }: { viewModeChanged?: boolean } = {}) => {
       if (!viewModeChanged || !canvasElement) return;
```

Comment:
does it need something like this?

```
if(this.torndown) return
```

Reply:
or to throw an error?

Reply:
Hmm, we didn't have that before and we don't "need" it as we don't ever teardown >1. But throwing an error unexpected code paths isn't the worst thing to do. I'll add something.

Reply:
Ok, I tried this and it turns out we do sometimes teardown more than once. I didn't really bother investigating why, it's not the end of the world to do so if it doesn't cause problems I don't think.

Reply:
SGTM